README.md

```diff
@@ -517,4 +517,137 @@ export class AuthResource extends AuthGuardResource {
 
 }
 
+```
+
+
+## Example of resource model usage
+
+
+### `UserResource with model`
+
+
+```ts
+import { Injectable } from '@angular/core';
+import { RequestMethod } from '@angular/http';
+import { AppProject } from '../../project/app.project';
+import { Resource, ResourceAction, ResourceMethod, ResourceParams, ResourceModelParams, ResourceModel } from 'ng2-resource-rest';
+import { SomeService } from './some.service'
+import { GroupResource, Group } from './group.resource'
+
+export interface IUserQueryInput {
+  is_active?: boolean;
+}
+
+export interface IUserShort {
+  id: number;
+  email: string;
+}
+
+export interface IUser extends IUserShort {
+  avatar: string;
+  first_name: string;
+  last_name: string;
+}
+
+export interface User extends IUser {}
```

Comment:
Why do you need to create User interface from IUser and then class with the same name?
I'm a bit confused or it's some mistake here?